backend/src/v2/compiler/argocompiler/common.go

```diff
@@ -46,6 +60,108 @@ var commonEnvs = []k8score.EnvVar{{
 	},
 }}
 
+var MLPipelineServiceEnv = []k8score.EnvVar{{
```

Comment:
Is this necessary? Kubernetes already populates these automatically. You can see `cacheDefaultEndpoint` already uses these.

@HumairAK thoughts?